### PR TITLE
Added resource type and changed filtered_resources

### PR DESCRIPTION
**Pull request by the-sinner**
## Description

Added resource type to the card and changed filtered_resources (Copy Link to Open Link)

## Screenshots (if any)
Screenshot 2023-02-04 at 11 31 26 PM

### CODE REVIEW

File: templates/resources.html [hunk 1]
```diff
@@ -50,6 +50,7 @@
                     <p class="title is-4">{{res.title}}</p>
                 </div>
                 <span class="tag is-danger">{{res.level}}</span>
+                <span class="tag is-danger">{{res.type}}</span>
```
Comment (Bhupesh-V) [inline templates/resources.html:53]: Do you think a different colour could make the standout, the type?
We don't really have a UX plan, but I believe a different colour could help in telling that a "type" is different from "level"

wdyt?

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]

File: templates/resources.html (hunk 1 above)
Comment (the-sinner) [reply templates/resources.html:53]: Yeah, i also think the same.
I tried some shades of blue that looked good with the reddish color.

1. #0075A2
2. #3273DC
3. #3E8ED0
Screenshot 2023-02-06 at 2 38 42 PM

Comment (Bhupesh-V) [reply templates/resources.html:53]: 3rd one looks perfect 👌🏽  `#3E8ED0`

Comment (the-sinner) [reply templates/resources.html]: I have changed the css in the new commit.

**Review by the-sinner — COMMENTED** [review]
Changed Copy link to Open link after filter is applied
Changed the resource type colour to blue.

**Comment by Bhupesh-V** [comment]
> Changed Copy link to Open link after filter is applied Changed the resource type colour to blue.

Cool, but it looks like your branch is not correctly updated from main, there are some unwanted changes, please rebase it or create a pr from another branch with your changes!

**Comment by the-sinner** [comment]
I have opened a new pull request #21 with the required changes.
I will be closing this pull request.

**Pull request closed without merging**